orchestration/Tests/UnitTests/ValidationResourceGroupSetup.Tests.ps1

```diff
@@ -0,0 +1,45 @@
+########################################################################################################################
+##
+## ValidationResourceGroup.Tests.ps1
+##
+##          The purpose of this script is to perform the unit testing for the ValidationResourceGroup Module using Pester.
+##          The script will import the ValidationResourceGroup Module and any dependency moduels to perform the tests.
+##
+########################################################################################################################
+
+
+. ../../OrchestrationService/ValidationResourceGroupSetup.ps1;
```

Comment:
Do not use fixed path, use Join-Path or [IO.Path]::Combine instead

Reply:
Fixed.